### PR TITLE
@mzikherman: Update Artsy Passport and small UI tweak in error handler

### PR DESCRIPTION
**Pull request by craigspaeth**
This brings in our small analytics/error handling updates in AP and makes a slight tweak to the error handler component to surface partial stack traces in a legible way.

### CODE REVIEW

**Comment by mzikherman** [comment]
Cool, I'll push a commit for those track calls, thanks!

File: mobile/analytics/auth.js [hunk 1]
```diff
@@ -38,9 +38,4 @@ if (Cookies.get('analytics-signup')) {
       user_id: sd.CURRENT_USER.id
     })
   }
-}
-
-// Successfully logged in
-analyticsHooks.on('auth:login', function(options) {
-  analytics.track('Successfully logged in');
-});
+};
```
Comment (mzikherman) [inline mobile/analytics/auth.js]: We need to make sure to end the file with a semi-colon yes?

Comment (craigspaeth) [inline mobile/analytics/auth.js]: I don't _think_ so because they're packaged with Browserify & Babelify—but doesn't hurt.

**Comment by mzikherman** [comment]
Alright, mergin'!

**Pull request merged**